### ISSUE TITLE
Sub-menu items under CMS menu not opening directly.

### ISSUE THREAD

**Issue by mjauvin** [wintercms/wn-tailwindui-plugin#6]
When opening the CMS menu with the arrow, if a sub-menu items is clicked (e.g. Content), the Pages sub-menu is opened instead.

Subsequent clicks on sub-menu items work as expected.

**Comment by LukeTowers** [comment]
Yeah, it's a known issue with all three of the menus that use that variation of on page JS routing (i.e. CMS, Pages, & Builder). The current solution is good enough for my needs so I'm probably not going to prioritize this (although I do need to add a snippets button to the richeditor because the snippets menu for the pages plugin is completely broken at the moment); but if someone else wants to they're more than welcome!

**Comment by LukeTowers** [comment]
Minor update: Snippets menu now works for Winter.Pages
